src/Kunstmaan/AdminBundle/Resources/config/services.yml

```diff
@@ -88,6 +88,10 @@ services:
         class: Kunstmaan\AdminBundle\Helper\CloneHelper
         arguments: ['@doctrine.orm.entity_manager', '@event_dispatcher']
 
+    kunstmaan_admin.isadminroute.helper:
+        class: Kunstmaan\AdminBundle\Helper\IsAdminRouteHelper
+        arguments: ['%admin_url%']
```

Comment:
How `'%admin_url%` is set? What if we configure it via [bundle configuration](http://symfony.com/doc/current/bundles/configuration.html)?